### PR TITLE
[DSN-2][DSN-3][DSN-4][DSN-13] Project setup

### PR DESCRIPTION
**Pull request by Kemoke**

### CODE REVIEW

**Review by a42m — CHANGES_REQUESTED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -2,6 +2,28 @@
 # yarn lockfile v1
```
[the remark this answers is not in the document]
Comment (Kemoke) [reply yarn.lock:2]: Agreed to defer this to a later time.

File: .storybook/preview.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import { app } from '@storybook/vue3'
+import DesignSystemNext from '../lib/index'
```
[the remark this answers is not in the document]
Comment (Kemoke) [reply .storybook/preview.js:2]: Aliases don't seem to work in this file.

[the remark this answers is not in the document]
Comment (Kemoke) [reply .storybook/preview.js:2]: Nevermind it works.

**Review by a42m — APPROVED** [review]

File: .stylelintrc [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "plugins": ["stylelint-prettier"],
+  "rules": {
+    "prettier/prettier": [true, {"singleQuote": true, "tabWidth": 4, "endOfLine": "auto"}]
```
[the remark this answers is not in the document]
Comment (kurent) [reply .stylelintrc:4]: yes, agreed

**Pull request merged**